sklearn/utils/discovery.py

```diff
@@ -41,6 +41,28 @@ def all_estimators(type_filter=None):
     estimators : list of tuples
         List of (name, class), where ``name`` is the class name as string
         and ``class`` is the actual type of the class.
+
+    Examples
+    --------
+    >>> from sklearn.utils.discovery import all_estimators
+    >>> estimators = all_estimators() # get all estimators
+    >>> type(estimators)
+    <class 'list'>
+    >>> type(estimators[0])
+    <class 'tuple'>
+    >>> list(dict(estimators))[:3]
+    ['ARDRegression', 'AdaBoostClassifier', 'AdaBoostRegressor']
```

Comment:
Let's no do any transform. Let's only show the 2 first element of what is returned.